sqlparser/dialect/mysql/dialect.go

```diff
@@ -21,8 +26,8 @@ func (dialect *MySQLDialect) QuoteHandler() dialect.QuoteHandler {
 }
 
 // NewMySQLDialect return new MySQLDialect
-func NewMySQLDialect() *MySQLDialect {
-	return &MySQLDialect{}
+func NewMySQLDialect(caseSensitiveTableName bool) *MySQLDialect {
```

Comment:
lets extend dialect configuration with options? case sensitivity is first option but probably not last. Lets simplify  future extending with dynamic amount of parameters? as we did it here - https://github.com/cossacklabs/acra/blob/master/decryptor/base/accessContext.go#L49